docs/gc-ms-deconvolution.md

```diff
@@ -33,11 +33,18 @@ The GNPS home page includes two sections to launch GNPS GC-MS data analysis job,
 - All of the data used for the job must be collected using the same GC protocol (temperature program, column, injection mode). Multiple datasets could be combined as long as the data have been collected using the same protocol. 
 - When creating a dataset on MassIVE ([massive.ucsd.edu](https://massive.ucsd.edu/ProteoSAFe/static/massive.jsp)), include “GNPS” and “GC” in the name (As GNPS_GC_Name of the GNPS-MassIVE dataset). See the documentation about [Dataset Creation/Sharing](https://ccms-ucsd.github.io/GNPSDocumentation/datasets/) and an example of a MassIVE dataset [MSV000084226](https://gnps.ucsd.edu/ProteoSAFe/result.jsp?task=671cd79ac3af4c4493e4025d62d161e1&view=advanced_view) for more information.
 - An insufficient volume of data causes unstable deconvolution resulting in spurious low-quality spectra. It is recommended for datasets smaller than ~10 files to use alternative deconvolution solutions such as MZMine2/ADAP, MS-Dial or XCMS. Increasing the number of files, for example, could be done by co-analyzing multiple datasets obtained with the same experimental protocol. 
+- MSHub generates the parameter that is termed “balance score”. The balance score quantifies reproducibility of the deconvoluted fragmentation patterns across the data (when it is high, the deconvoluted spectrum is consistent across different samples). The balance score also gives insight into how well the spectral feature is explained by the available data - a complete, fully characterized fragmentation pattern of a real compound would be the same in different measurements, while noisy spectra with missing or chimeric peaks would change randomly across files, resulting in a low balance score. The balance score is independent of the number of files in the dataset, and even when a compound is present in only a few samples in the dataset, as long as the spectral patterns, irrespective of compound abundances, are conserved across samples (e.g. not missing peaks or contaminated by spurious noise peaks), it would result in a high balance score.
 
 
 
 The **“Advanced MSHub Processing” section** displays some automatic setting parameters (related to peak symmetry and baseline adjustment) determined internally by MSHub. The user does not need to provide settings, although they can be set manually, should the user choose to do so. 
 
+The adjustable settings are:
```

Comment:
Do you know if we actually use these? I thought I disabled them in the actual code.

Reply:
you're right, the user cannot choose to change parameters for Ms-Hub, but Alexander asked me to add that so I was thinking that you planned to add this option.

Reply:
maybe a reviewer asked for that, I don't have the explaination but I can ask to Alexander.